README.md

```diff
@@ -119,6 +119,39 @@ Once in the web app:
 * Explore citations and sources
 * Click on "settings" to try different options, tweak prompts, etc.
 
+## Automate deployment
+
+You can use GitHub actions or Azure DevOps to deploy this demo. For example, after pushing changes to the repo.
+
+There is a `.github` and `.azdo` folder which provides a pipeline (or workflow) definition (`azure-dev.yml`). These pipelines require some configuration for the provider (GitHub or Azure DevOps), for example, setting secrets or variables. You can use `azd` to create the configuration for the provider:
+
+For `GitHub`, run:
+> azd pipeline config
+
+For `Azure DevOps`, run: 
+> azd pipeline config --provider azdo
+
+Notes:
+
+- Enable the `GitHub workflow` to provision and deploy by renaming the file `azure-dev.yml.disabled` to `azure-dev.yml`.
+
+- Create an entry in `main.paramenters.json` for `openAiResourceGroupLocation` like:
+```json
+  "parameters": {
+    "openAiResourceGroupLocation":
+    {
+      "value": "LOCATION"
+    }
+  }
+```
+Use a valid location for `LOCATION`.
```

Comment:
We could give them more details like "If you've already deployed manually, use the location you selected at the prompt. If not, make sure you specify one of the [supported locations](https://learn.microsoft.com/azure/cognitive-services/openai/concepts/models#model-summary-table-and-region-availability)."